WordPress/Classes/Models/Blog.m

```diff
@@ -400,6 +402,19 @@ - (NSSet *)allowedFileTypes
     return [NSSet setWithArray:allowedFileTypes];
 }
 
+- (void)setOptions:(NSDictionary *)options
+{
+    [self willChangeValueForKey:@"options"];
+    [self setPrimitiveValue:options forKey:@"options"];
+    self.jetpack = nil;
+    [self didChangeValueForKey:@"options"];
+}
+
++ (NSSet *)keyPathsForValuesAffectingJetpack
```

Comment:
It seems like this method is not called from anywhere.  If it needs to stay for future changes I'd document this method and explain what "values affecting jetpack" means.